### ISSUE TITLE
Death Knight T10 4p Bonus not proccing in all conditions

### ISSUE THREAD

**Issue by tripleslash** [dalaranwow/dalaran-wow#4870]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: When placing a Death and Decay and using Army of the Dead, all runes are on cooldown but the DK does not gain the buff "Advantage" (T10 4p).

**Expected behaviour**: The 4p bonus reads: Whenever all your runes are on cooldown, you gain 3% increased damage done with weapons, spells and abilities for the next 15 sec.

**Steps to reproduce the problem**:

1. Summon Army
2. Use Death and Decay
3. Check your buffs and notice you don't have "Advantage" up.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @tripleslash, thank you for the report.
I can confirm the issue on ptr.
 I would like to add to the report that the DK doesn't get the 3% damage increase spell https://wowgaming.altervista.org/aowow/?spell=70656 only when he is out of combat (while in combat it is always applied).

**Comment by TeridanDW** [comment]
Works as intended on PTR.
